shema_based_on_diagram.sql

```diff
@@ -0,0 +1,36 @@
+CREATE TABLE patients (
+  id BIGSERIAL NOT NULL PRIMARY KEY,
+  name VARCHAR(100),
+  date_of_birth DATE
+);
+CREATE TABLE medical_histories (
+  id BIGSERIAL NOT NULL PRIMARY KEY,
+  admitted_at TIMESTAMP,
+  patient_id BIGINT NOT NULL REFERENCES patients(id),
+  status VARCHAR(100)
+);
+
```

Comment:
- [ ] Please add FOREIGN KEYS indexes as requested in the project requirements. hint:
![image](https://user-images.githubusercontent.com/83803414/181572178-642f7859-cfc6-403b-8207-0d8924024f37.png)